broker/scripts/bridge_functions.sh

```diff
@@ -16,8 +16,6 @@ ensure_bridge()
     ip addr add 10.254.0.2/16 dev $brname
     # TODO Policy routing should probably not be hardcoded here?
     ensure_policy from all iif $brname lookup mesh prio 1000
-    # Disable forwarding between bridge ports
-    ebtables -A FORWARD --logical-in $brname -j DROP
```

Comment:
You have to do the isolation per port, and isolating the bridge interface itself would mean that none of the attached bridge porst would be able to communicate with host.

Reply:
@pmelange is correct. `isolation on` is a function of the bridge port, not the bridge itself. As such it would functionally belong into the session handler scripts.